mbuild/compound.py

```diff
@@ -43,8 +43,8 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
 
     Parameters
     ----------
-    filename : str
-        Name of the file from which to load atom and bond information.
+    filename_or_topology : str
```

Comment:
what about `filename_or_object`? `topology` can mean a handful of different things and I think `object` refers more broadly to the different Python objects in memory that we support or will support in the future.

Reply:
Sure thing, this should be a small fix.